appveyor.yml

```diff
@@ -67,20 +63,23 @@ deploy:
       branch: dev
 
   - provider: GitHub
+    repository: Flow-Launcher/Prereleases
     release: v$(flowVersion)
+    description: 'This is a release candidate of v$(flowVersion)'
     auth_token:
       secure: ij4UeXUYQBDJxn2YRAAhUOjklOGVKDB87Hn5J8tKIzj13yatoI7sLM666QDQFEgv
     artifact: Squirrel Installer, Portable Version, Squirrel nupkg, Squirrel RELEASES
-    draft: true
     force_update: true
     on:
       branch: master
+      APPVEYOR_REPO_TAG: false
```

Comment:
Is this needed? Does this mean trigger on master or non-tagged builds?

Reply:
What I want is for it to trigger on master & tag, but I need to test to make sure that's how it behaves

Reply:
If I am not mistaking, I think you are trying to cover a scenario (doing a hot fix during a release) that has never happened. We have had several times where we needed to immediately rollout a hot fix after a release, but never during.

Assuming a tag is only created after setting the draft release to latest, you can just cater for:
1. Merge to master -> draft release, so 'on master' condition
2. Setting draft release to latest -> trigger 'on tag' -> publish NuGet and Prerelease

I think this is a simpler pipeline process. Any hot fixes we just do a new release. If we need to really hot fix during a release, I can just delete draft release, then merge hot fix in, which will kick off the above process again.

You will find testing this is a painful process because our build & test takes a while, recommend fork the project and the prerelease project, comment out the unit test code and test that way. Also I might be wrong but off the top of my head I think the 'on' condition you got is 'master' or 'tag is false'.

Reply:
> If I am not mistaking, I think you are trying to cover a scenario (doing a hot fix during a release) that has never happened. We have had several times where we needed to immediately rollout a hot fix after a release, but never during.

I am not trying to cover that scenario, really... it just happens to be possible with this setup, so I'm spelling it out.

> Assuming a tag is only created after setting the draft release to latest, you can just cater for:
> 
>     1. Merge to master -> draft release, so 'on master' condition
> 
>     2. Setting draft release to latest -> trigger 'on tag' -> publish NuGet and Prerelease

so you suggest that we push to Prereleases when `APPVEYOR_REPO_TAG: true` right? 

> Also I might be wrong but off the top of my head I think the 'on' condition you got is 'master' or 'tag is false'.

yeah it could be so, I'll define a different env variable for this to be sure

Reply:
>so you suggest that we push to Prereleases when APPVEYOR_REPO_TAG: true right?

Yeah. I don't think it matters if it's pushed on merge to master or when APPVEYOR_REPO_TAG: true. If on merge to master you can set force push to true so hot fix during a release will still update the prerelease artefacts.

Reply:
release artifacts wont be exactly the same but that's no big deal